changes/201.added

```diff
@@ -0,0 +1 @@
+- New CLI arguments to Camelyon benchmark (`--torch-gpu` and `--cp-name`)([#201](https://github.com/Substra/substrafl/pull/201))
```

Comment:
I think there is a mismatch with the PR number (203) and the one you put here (201)

Reply:
It's not a change introduced by **this** PR, it was backporting previous changes so that they'll appear in the next release.

Reply:
Ah yes of course, my bad I didn't read the content 😅